### PR TITLE
[#2688] Vuestic bundle size statistics script

### PR DESCRIPTION
**Pull request by aluarius**
Closes: #2688 

## Description
- [x] script to calculate Vuestic & VaButton size in project bundle after treeshaking;
- [x] docs update.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Review by rustem-nasyrov — APPROVED** [review]
Great job!

**Comment by aluarius** [comment]
image

image

@asvae Explanation of the our bundle size is that our code is highly reusable. If 1-2 components are used - it takes more space in the project bundle than other UI-kits. But if 3-5 components are used, we become much more attractive in this question. For example, VaButton and VaSelect bundle contains:

image

File: packages/sandbox/dev-tree-shaking.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (m0ksem) [inline packages/sandbox/dev-tree-shaking.md:1]: Looks cool.

**Comment by m0ksem** [comment]
On my computer, bundle size so much lower :) 

image

Can we automate this thing using build plugin?

**Review by m0ksem — APPROVED** [review]

**Pull request merged**